packages/devtools_app/lib/src/screens/performance/panes/raster_stats/raster_stats.dart

```diff
@@ -21,12 +21,21 @@ class RasterStatsView extends StatelessWidget {
   const RasterStatsView({
     super.key,
     required this.rasterStatsController,
+    required this.impellerEnabled,
   });
 
   final RasterStatsController rasterStatsController;
 
+  final bool impellerEnabled;
+
   @override
   Widget build(BuildContext context) {
+    if (impellerEnabled) {
+      return const CenteredMessage(
+        'The Raster Stats tool is not yet ready for the Impeller rendering '
```

Comment:
I would adjust the phrasing here to something like "not currently available", which implies that it may change but I think doesn't imply as much urgency.